### PR TITLE
GPU backend cell_idx fix (test needed) + JOSS paper bib corrections

### PR DESCRIPTION
**Pull request by edejong-caltech**
To (hopefully) address issue in PR#1015.

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -310,7 +310,7 @@ Development of ice-phase microphysics representation has been supported through
   grant no. DE-SC0021034 by the Atmospheric System Research Program and 
   Atmospheric Radiation Measurement Program sponsored by the U.S. Department of Energy (DOE).
 EdJ's contributions were made possible by support from the Department of Energy Computational Sciences Graduate Research Fellowship.
-SAr, OB and KD acknowledge support from the Polish National Science Centre (grant no. 2020/39/D/ST10/01220).
+SAr acknowledges support from the Polish National Science Centre (grant no. 2020/39/D/ST10/01220).
```
Comment (slayoo) [inline paper/paper.md:313]: @edejong-caltech this change reverts a valid commit

File: PySDM/backends/impl_thrust_rtc/methods/collisions_methods.py [hunk 2]
```diff
@@ -481,10 +481,10 @@ def __init__(self):
         )
 
         self.__normalize_body_1 = trtc.For(
-            param_names=("prob", "cell_id", "norm_factor"),
+            param_names=("prob", "cell_idx", "cell_id", "norm_factor"),
```
Comment (slayoo) [inline PySDM/backends/impl_thrust_rtc/methods/collisions_methods.py:484]: @edejong-caltech let's add a minimal test for this one!

**Review by slayoo — CHANGES_REQUESTED** [review]

File: PySDM/backends/impl_thrust_rtc/methods/collisions_methods.py (hunk 2 above)
Comment (edejong-caltech) [reply PySDM/backends/impl_thrust_rtc/methods/collisions_methods.py:484]: Perhaps you can help suggest a test @slayoo ?

**Comment by slayoo** [comment]
@edejong-caltech @abulenok 

IIUC, this PR is related with https://github.com/open-atmos/PySDM/issues/330

Currently, adaptivity in coalescence is only supported on GPU for zero-dimensional simulations, where the `cell_idx` is a one-element zero-filled array, hence skipping it is OK. I'd say that we should have a comprehensive solution introducing adaptivity on GPU for >0D simulations rather than just fixing it in one selected place without a test as done herein. Also, the JOSS paper updates introduced in this PR are actually reverting meaningful changes in the main branch.

Closing.

**Pull request closed without merging**